article/test/ArticleFeatureTest.scala

```diff
@@ -599,44 +598,5 @@ import collection.JavaConverters._
         $(".content").attribute("class") should include("tone-comment")
       }
     }
-
-    scenario("Display breadcrumbs correctly") {
```

Comment:
We don't have something called breadcrumbs anymore, but there are tests here instead: https://github.com/guardian/frontend/blob/master/common/test/navigation/NavigationTest.scala